openlibrary/plugins/upstream/adapter.py

```diff
@@ -105,7 +105,10 @@ def process_error(self):
                 d['key'] = unconvert_key(d['key'])
             self.output = simplejson.dumps(d)
 
-def convert_key(key, mapping=convertions):
+
```

Comment:
Please undo the changes to this file.

Reply:
Okay

Reply:
Please remove lines 108 and 125 so there are no changes to this file.